CHANGELOG.md

```diff
@@ -6,12 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.7.2]
+### Added
```

Comment:
This changelog needs to be fixed; we have two empty sections and we're missing a change entry

Reply:
Hey @Gudahtt i believe this list is still WIP but i have included the latest changes here [#1a6d367a](https://github.com/MetaMask/metamask-extension/pull/22336/commits/1a6d367a298eff2ea6d18381c2cbc74b82fced12)  please take a look at it and let me know if there's any issues.